### ISSUE TITLE
type errors

### ISSUE THREAD

**Issue by veeramarni** [Tyriar/node-pty#100]
```
lerna ERR! execute ERROR in ./src/terminalProcess.ts
lerna ERR! execute (60,12): error TS2339: Property 'on' does not exist on type 'ITerminal'.
lerna ERR! execute
lerna ERR! execute ERROR in ./src/terminalProcess.ts
lerna ERR! execute (71,12): error TS2339: Property 'on' does not exist on type 'ITerminal'.
lerna ERR! execute
lerna ERR! execute ERROR in ./src/terminalProcess.ts
lerna ERR! execute (135,25): error TS2339: Property 'pid' does not exist on type 'ITerminal'.
```

I was testing based on the file https://github.com/Microsoft/vscode/blob/3e4a7d1e89d61f3687926f1f8072e9cabd3c781b/src/vs/workbench/parts/terminal/node/terminalProcess.ts

**Comment by Tyriar** [comment]
@veeramarni what are you running to get the errors?

**Comment by Tyriar** [comment]
Duplicate of https://github.com/Tyriar/node-pty/issues/105, fixed in 0.6.9
